tests/Microsoft.Identity.Web.Test/PrincipalExtensionsForSecurityTokensTests.cs

```diff
@@ -0,0 +1,73 @@
+using Microsoft.IdentityModel.JsonWebTokens;
+using Microsoft.IdentityModel.Tokens;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Security.Principal;
+using Xunit;
+
+namespace Microsoft.Identity.Web.Tests
+{
+    public class PrincipalExtensionsForSecurityTokensTests
+    {
+        [Fact]
+        public void GetBootstrapToken_Returns_Null_When_ClaimsPrincipal_Is_Null()
+        {
+            // Arrange
+            IPrincipal claimsPrincipal = null;
+
+            // Act
+            var result = claimsPrincipal.GetBootstrapToken();
+
```

Comment:
there are a few warnings in the test to address

Reply:
Done. Thanks!